source/grid/grid_tools.inst.in

```diff
@@ -18,7 +18,8 @@ for (X : TRIANGULATIONS; deal_II_dimension : DIMENSIONS;
   {
 #if deal_II_dimension <= deal_II_space_dimension
     namespace GridTools
-    \{
+
+    {
       template std::pair<
```

Comment:
There seem to be a lot spurious changes in this file.

Reply:
Sorry about this. I have fixed it!